### PR TITLE
Adding WhatsApp class

### PR DESCRIPTION
**Pull request by hfingler**
Added, WhatsApp scss and less class, added to available icons on website (not button block because I didn't want to change the standard "Log in with X" to a mobile number.

### CODE REVIEW

**Comment by vburlak** [comment]
@hfingler and what should be done by clicking WhatsApp button? :smirk:

**Comment by hfingler** [comment]
@vburlak Exactly, that's why I didn't add the long button for the WhatsApp, only the small icon. It is useful (at least for Brazilian users, because we use WhatsApp for everything, even customer service) for showing your contact number (I put the number as image instead of text):

captura de tela 2015-07-08 as 14 16 21

 Anyway, I bet Facebook is going to integrate their Web WhatsApp with Facebook or make links to automatically add people to your WhatsApp contacts soon. Maybe put this on hold until then?

 Thanks.

File: bootstrap-social.scss [hunk 1]
```diff
@@ -112,3 +112,4 @@ $bs-height-xs:   (floor($font-size-small * 1.2) + $padding-small-vertical + 1);
 .btn-vimeo         { @include btn-social(#1ab7ea); }
 .btn-vk            { @include btn-social(#587ea3); }
 .btn-yahoo         { @include btn-social(#720e9e); }
+.btn-whatsapp      { @include btn-social(#439520); }
```
Comment (lipis) [inline bootstrap-social.scss:115]: where did you get this color?! source?

Comment (hfingler) [inline bootstrap-social.scss:115]: I got it from the WhatsApp symbol here:

http://media.whatsapp.com/logos/Vector

Comment (Jonathan-Rosewood) [inline bootstrap-social.scss:115]: Hm...i see gradient on this logos :)

**Pull request closed without merging**